images/manageiq-ui-worker/Dockerfile

```diff
@@ -6,16 +6,22 @@ MAINTAINER ManageIQ https://manageiq.org
 
 ARG NPM_REGISTRY_OVERRIDE
 
+ARG ARCH=amd64
+
 LABEL name="manageiq-ui-worker" \
       summary="ManageIQ user interface worker image"
 
 # Install httpd and remove the existing httpd config from manageiq-appliance
-RUN yum -y install httpd --setopt=tsflags=nodocs && \
+RUN yum -y install httpd sudo --setopt=tsflags=nodocs && \
```

Comment:
Don't think we should need sudo during the build.

Reply:
removed, it was for executing the phantomjs script which was created to run in non-root user mode.